src/gui/joystickstatuswindow.cpp

```diff
@@ -336,9 +338,9 @@ void JoystickStatusWindow::updateAccelerometerValues(float valueX, float valueY,
  */
 void JoystickStatusWindow::updateGyroscopeValues(float valueX, float valueY, float valueZ)
 {
-    m_gyro_axes[0]->setValue(JoySensor::radToDeg(valueX) * 1000);
-    m_gyro_axes[1]->setValue(JoySensor::radToDeg(valueY) * 1000);
-    m_gyro_axes[2]->setValue(JoySensor::radToDeg(valueZ) * 1000);
+    m_gyro_axes[0]->setValue(valueX * 1000);
+    m_gyro_axes[1]->setValue(valueY * 1000);
+    m_gyro_axes[2]->setValue(valueZ * 1000);
 }
```

Comment:
Why did you remove these conversions?

Reply:
Oops. I commented out those values, then tried a lot of things to get it working, and to undo it I ended up just copying the conversions from above to revert the changes.

Thanks for spotting that. I've reverted that chunk.